eng/common/scripts/Update-ChangeLog.ps1

```diff
@@ -0,0 +1,109 @@
+# Note: This script will add or replace version title in change log
+
+# Parameter description
+# Version : Version to add or replace in change log
+# ChangeLogPath: Path to change log file. If change log path is set to directory then script will probe for change log file in that path 
+# Unreleased: Default is true. If it is set to false, then today's date will be set in verion title. If it is True then title will show "Unreleased"
+# ReplaceVersion: This is useful when replacing current version title with new title.( Helpful to update the title before package release)
+
+param (
+  [Parameter(Mandatory = $true)]
+  [String]$Version,
+  [Parameter(Mandatory = $true)]
+  [String]$ServiceDirectory,
+  [Parameter(Mandatory = $true)]
+  [String]$PackageName,
+  [boolean]$Unreleased=$True,
+  [boolean]$ReplaceVersion = $False
+)
+
+DynamicParam {
```

Comment:
What is a DynamicParam? Why is this needed?

Reply:
I guess it not necessary. But I was trying to make sure you can only pass the `ReleaseDate` parameter when Unreleased is set to $False. That eill make other part of the script more straight forward.